### PR TITLE
RUMM-413 Make envrionment property mandatory for initialisation

### PR DESCRIPTION
**Pull request by mariusc83**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

### CODE REVIEW

File: dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogConfig.kt [hunk 1]
```diff
@@ -181,7 +191,8 @@ private constructor(
 
         /**
          * Sets the environment name that will appear in your logs, traces and crash reports.
-         * This can be used to filter logs or traces and distinguish between your production and staging environment.
+         * This can be used to filter logs or traces and distinguish between your production
+         * and staging environment.
          * @param envName the environment name (default = "")
          */
         fun setEnvironmentName(envName: String): Builder {
```
Comment (ncreated) [inline dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogConfig.kt:198]: Shouldn't this be marked as deprecated now, as we expect `envName` in `Builder's` constructor?

Comment (mariusc83) [reply dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogConfig.kt:198]: Not really sure...maybe. I will raise this in the daily

File: instrumented/benchmark/gradle.properties [hunk 2]
```diff
@@ -1 +1 @@
-android.enableAdditionalTestOutput=true
\ No newline at end of file
+android.enableAdditionalTestOutput = true
```
Comment (xgouchet) [inline instrumented/benchmark/gradle.properties:1]: Why did you change this ? Not sure it'll keep working everywhere :/

Comment (mariusc83) [reply instrumented/benchmark/gradle.properties:1]: I didn't...maybe it was ktlint.....I will put it back ;)
